scripts/combine/setupCombine.py

```diff
@@ -603,6 +604,16 @@ def setup(args, inputFile, fitvar, xnorm=False):
             labelsByAxis=["unc_ieta", "unc_ipt", "downUpVar"] if not correlated_non_closure else ["downUpVar"],
             passToFakes=passSystToFakes
         )
+    if not input_tools.args_from_metadata(cardTool, "noSmearing"):
+        cardTool.addSystematic("muonResolutionSyst_responseWeights", 
+            mirror = True,
+            processes=['single_v_samples'],
+            group="resolutionCrctn" if args.sepImpactForNC else "muonScale",
```

Comment:
I think we want this in a separate group, it's not really part of the scale and having it grouped together by default complicates the interpretation

Reply:
Oh, I see you introduced sepImpactForReso, so I assume you meant to use that here. Regardless I don't think it's needed, we should eventually restructure the setup so that we have a more consistent interplay between fine grained and course groups.

Reply:
I think the reason we'd like to have the option to keep the resolution uncertainty in the muonScale group is that we want to study whether (massfit stats. unc. + non-closure nuisances + resolution) is sufficient to cover the non-closure bias.

Reply:
Best would be to have both a full group named muonCalibration and the separated nuisance. this can be done following PR #308. We can merge this first, but you should still fix the bug where sepImpactForNC is used instead of sepImpactForNonclos

Reply:
I've always been using `sepImpactForNC`. Where is `sepImpactForNonclos` used? I think that needs to be fixed.

Reply:
Sorry, I mean sepImpactForReso. You've introduced it but I don't see it used.

Reply:
ah, that is used for the fit numbers I showed.
If `--sepImpactForReso` is not enabled, the resolution impact will be in the muonScale group.
I agree that we should probably use a more generic group name like muonCalibration, I will change that.

Reply:
sepImpactForReso for isn't used anywhere as far as I can tell. Here you use args.sepImpactForNC

Reply:
oops, you are right, that slipped my mind. Thanks for catching that!